packages/ckeditor5-paste-from-office/src/filters/image.ts

```diff
@@ -80,9 +80,18 @@ function findAllShapesIds( documentFragment: ViewDocumentFragment, writer: Upcas
 		const el = value.item as ViewElement;
 		const previousSibling = el.previousSibling;
 		const prevSiblingName = previousSibling && previousSibling.is( 'element' ) ? previousSibling.name : null;
+		// If shape element has 'o:gfxdata' attribute and is not directly before
+		// `<v:shapetype>` element it means that it represents a Word shape.
+		// https://github.com/ckeditor/ckeditor5/pull/15847#issuecomment-1941543983
+		const exceptionIds = [ 'Chart' ];
 
 		// If shape element have 'o:gfxdata' attribute and is not directly before `<v:shapetype>` element it means it represent Word shape.
-		if ( shapeElementsMatcher.match( el ) && el.getAttribute( 'o:gfxdata' ) && prevSiblingName !== 'v:shapetype' ) {
+		if (
+			shapeElementsMatcher.match( el ) &&
+			el.getAttribute( 'o:gfxdata' ) &&
+			prevSiblingName !== 'v:shapetype' &&
+			!exceptionIds.some( item => el.getAttribute( 'id' )!.includes( item ) )
+		) {
```

Comment:
The `if` statement becomes too big and it's almost unreadable without analyzing each condition. Lets make it better to read :book: 

I suggest to extract every condition to a `const` and name it this way so the comment above won't be needed.

Suggestion code:

```
const isElementAShape = shapeElementsMatcher.match( el );
const hasElementGfxdataAttribute = el.getAttribute( 'o:gfxdata' );
const isPreviousSiblingAShapeType = prevSiblingName === 'v:shapetype';
const isElementIdInExcpetionsArray = exceptionIds.includes( el.getAttribute('id')! );

if (
	isElementAShape &&
	hasElementGfxdataAttribute &&
	!isPreviousSiblingAShapeType &&
	!isElementIdInExcpetionsArray
)
```

I've change also `some` to `includes` array method - also should increase the readability.